examples/hpc-enterprise-slurm.yaml

```diff
@@ -95,9 +99,10 @@ deployment_groups:
 
   - id: projectsfs
     source: modules/file-system/filestore
-    use: [network1]
+    use: [network1, private-service-access]
     settings:
       local_mount: /projects
+      connect_mode: PRIVATE_SERVICE_ACCESS
```

Comment:
I am late to the party, but was it considered that this variable could be passed via use?

Reply:
I don't think both values for that variable could be passed via `use`.  You are semi-correct because in the blueprint as it stands `connect_mode: PRIVATE_SERVICE_ACCESS` is redundant because by passing the psa module into the filestore it will automatically set the connect mode in the module.  I left it in so that it was apparent to users what was happening.

That being said, if a user has a PSA already set up in their account you still need `connect_mode` to allow both options if no psa module is in the blueprint with the filestore module.